Reformulate instructions to use lalsuite from source (#459)
PyFstat/PyFstat: README.md

### Using LALSuite built from source

> Instructions to use a custom local LALSuite installation can be found in [this wiki page](https://github.com/PyFstat/PyFstat/wiki/Using-LALSuite-built-from-source).

Reviewer: to satisfy readthedoc's unique link name requirement:
```suggestion
Instructions to use a custom local LALSuite installation can be found in [here on the wiki](https://github.com/PyFstat/PyFstat/wiki/Using-LALSuite-built-from-source).
```

[from source](https://git.ligo.org/lscsoft/lalsuite/),
make sure it is **swig-enabled** and contains at least the `lal` and `lalpulsar` packages.

LALsuite should be compiled *after creating a virtual environment*

> [as strongly recommended earlier in this document](#installation),

Author: No, since lalsuite is directly installed on the environment itself via the `config` flag.

> [as strongly recommended earlier in this document](#installation),

Author: False, see wiki.
